test/common_framework.py

```diff
@@ -116,8 +116,16 @@ def setUp(self):
 
         # create minimal startup files
         mkfile(self.confsys, """\
+SWITCHES=/F\r
+DOS=UMB,HIGH\r
 lastdrive=Z\r
-device=dosemu\emufs.sys\r
+files=40\r
+stacks=0,0\r
+buffers=10\r
+device=dosemu\\emufs.sys\r
+device=dosemu\\umb.sys\r
+devicehigh=dosemu\\ems.sys\r
+devicehigh=dosemu\\cdrom.sys\r
 """)
```

Comment:
Hi,
   You if you want multiline file content to not break indentation in your python code there is dedent `textwrap.dedent()`, e.g. 

```python
    MULTILINE_CONTENT = dedent("""\
        @echo off
        echo "Some multiline batch file"
        """)
```

Reply:
Thanks, that's interesting. As I'm sure you've noticed the current pattern is repeated all over the place! I'll try it out.

Reply:
I've added it to the latest commit, along with the use of the newline option to `open()` to avoid sprinkling the carriage returns in the string.